README.adoc

```diff
@@ -1,49 +1,33 @@
 = OpenShift Documentation
 Welcome to the OpenShift documentation GitHub repository that contains the source files for the technical documentation for the following products:
-
 *IMPORTANT*
 The information in this README file and the associated contribution guide applies to only OpenShift Container Platform version 3.
-
 * https://www.okd.io/[OKD]
 * https://www.openshift.com/products/online/[OpenShift Online]
 * https://www.openshift.com/products/container-platform/[OpenShift Container Platform]
 * https://www.openshift.com/products/dedicated/[OpenShift Dedicated]
-
 All OpenShift documentation is sourced in http://www.methods.co.nz/asciidoc/[AsciiDoc] and transformed into HTML/CSS and other formats through automation that is based on http://asciidoctor.org/[AsciiDoctor].
-
 The documentation published from these source files can be viewed at http://docs.openshift.com.
-
 == Contributing to OpenShift documentation
 If you are interested in contributing to OpenShift technical documentation, you can view all our link:./contributing_to_docs[resources] that will help you get set up and provide more information.
-
-
 The following table provides quick links to help you get started.
-
 [options="header"]
 |===
-
 |Question |Link
-
 |I'm interested, how do I contribute?
 |See the link:/contributing_to_docs/contributing.adoc[contributing] topic to learn more about this repository and how you can contribute.
-
 |Are there any basic guidelines to help me?
 |The link:/contributing_to_docs/doc_guidelines.adoc[documentation guidelines] topic provides some basic guidelines to help us keep our content consistent, and includes other style information.
-
 |How do I set up my workstation?
 |See the link:/contributing_to_docs/tools_and_setup.adoc[tools and setup] topic to set up your workstation.
-
 |How do I edit an existing topic, or create new content?
 |See the link:/contributing_to_docs/create_or_edit_content.adoc[create or edit content] topic to get started.
-
 |What do I need to know about contributing content for 4.x?
 |See the link:/contributing_to_docs/contributing_user_stories.adoc[contributing user stories] topic to learn how the docs and contributing to the docs are changing in 4.0.
 |===
-
 == Contacts
-
 For questions or comments about OpenShift documentation:
-
 * OpenShift team members can be found on the http://webchat.freenode.net/?randomnick=1&channels=openshift&uio=d4[#openshift] and http://webchat.freenode.net/?randomnick=1&channels=openshift-dev&uio=d4[#openshift-dev] channels on http://www.freenode.net/[FreeNode].
 * You can also join the http://lists.openshift.redhat.com/openshiftmm/listinfo/users[Users] or http://lists.openshift.redhat.com/openshiftmm/listinfo/dev[Developers] mailing list.
 * Send an email to the OpenShift documentation team at openshift-docs@redhat.com.
+Collapse
```

Comment:
I was about to comment that there were merge conflict lines in here, but it looks like you're working on resolving those. 
What's left is this random "Collapse" and some removed lines (that are important for some of the formatting to properly render).

Reply:
Yeah, but it's all back as it should be. Line breaks back and no commit.